tools/cldr-code/src/main/java/org/unicode/cldr/util/UnitConverter.java

```diff
@@ -1617,6 +1620,26 @@ public static Set<UnitSystem> fromStringCollection(Collection<String> stringUnit
         public static Set<String> toStringSet(Collection<UnitSystem> stringUnitSystems) {
             return stringUnitSystems.stream().map(x -> x.toString()).collect(Collectors.toSet());
         }
+
+        private static final Joiner SLASH_JOINER = Joiner.on("/");
+
+        public static String getSystemsDisplay(Set<UnitSystem> systems) {
+            List<String> result = new ArrayList<>();
+            for (UnitSystem system : systems) {
+                switch (system) {
+                    case ussystem:
+                        result.add("US");
+                        break;
+                    case uksystem:
+                        result.add("UK");
+                        break;
+                    case jpsystem:
+                        result.add("JP");
+                        break;
+                }
```

Comment:
don't we already have localization for this field?

Reply:
These and the unit equalities are unlocalized (like the Code field)